src/render/vector.ts

```diff
@@ -14,6 +14,9 @@ export class Vector implements IPath {
     add(deltaX: number, deltaY: number): Vector {
         return new Vector(this.x + deltaX, this.y + deltaY);
     }
+    reverse(): Vector {
```

Comment:
Is it really necessary to create a function that just returns this?

Reply:
If this is not done, the border radius in canvas will not display correctly.